### ISSUE TITLE
[7.0.0] Keyboard navigation does not work on merged cells with hidden

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#6973]
### Description

Keyboard navigation with selection does not work on merged cells with hidden Row/columns.

### Steps to reproduce

1. Open demo https://jsfiddle.net/aninde/j412knaq/
2. Click on A2.
3. Keep pressing <kbd>cmd</kbd> + <kbd>Right arrow</kbd> or keep pressing arrow.
![selection-keyboard7 4 2](https://user-images.githubusercontent.com/30186818/83658681-d5f40900-a5c2-11ea-932f-49f4a57f895b.gif)

### Demo

https://jsfiddle.net/aninde/j412knaq/ - `v7.0.0`
https://jsfiddle.net/AMBudnik/dbcqm7vk/1/ `v7.4.2`

Actual result: selection after selecting the last cell returns to A2 and continues on the last row 4. 
Expected result: expected result: selection selects each cell over and over again

### Your environment
* Handsontable version: 7.0.0
* Browser Name and version: any
* Operating System: macOs Catalina 10.15.5

**Comment by aninde** [comment]
Fixed by `8.0.0-beta2` https://jsfiddle.net/AMBudnik/ktqogzwb/ rev10
![rev10-selection-keyboard](https://user-images.githubusercontent.com/30186818/83658775-f2904100-a5c2-11ea-9101-9678e091b209.gif)

**Comment by aninde** [comment]
I corrected the title because arrow navigation does not work regardless of selection.

**Comment by AMBudnik** [comment]
Updated demo https://jsfiddle.net/1v726ncz/ v 8.0.0. Issue fixed.
